runtime/config.yaml

```diff
@@ -30,4 +30,28 @@ api:
               "website_status": 1,
               "req_id": {{ .ReqID }}
             }
-      
\ No newline at end of file
+    - method: "chain"
```

Comment:
maybe I am missing something but how does the backend understand what dependency handling to use?
i.e. Aggregate or Chaining?

Reply:
@shan-96 method here stands for declaration of api call, basically if any of sub calls has declarations dependencies, then in this case they will be call with respect of dependencies. 

you can test it with these calls 

```
{"method":"chain", "req_id": 123, "passthrough":{"key":1}}
```

```
{"method":"aggregate", "params": {"country": "id"}, "req_id": 123, "passthrough":{"key":1}}
```